rerun_py/src/python_bridge.rs

```diff
@@ -792,6 +792,22 @@ impl PyMemorySinkStorage {
 
         self.inner.num_msgs()
     }
+
+    /// Drain all messages logged to the [`MemorySinkStorage`] and return as bytes.
+    ///
+    /// This will do a blocking flush before returning!
+    fn drain_as_bytes<'p>(&self, py: Python<'p>) -> PyResult<&'p PyBytes> {
+        // Release the GIL in case any flushing behavior needs to cleanup a python object.
+        py.allow_threads(|| {
+            self.rec.flush_blocking();
```

Comment:
Ha! So if you go through the bridge you do get a flush, but not if you're in pure Rust then.

So yeah I guess this can be removed and put directly into `MemorySinkStorage::drain_as_bytes`